localbuild.xml

```diff
@@ -0,0 +1,869 @@
+
+  <!--msm8998-->
+  <project path="device/sony/yoshino" name="device-sony-yoshino" groups="device" remote="sony" revision="r-mr1" />
+  <project path="device/sony/lilac" name="device-sony-lilac" groups="device" remote="sony" revision="r-mr1" />
+  <project path="device/sony/maple" name="device-sony-maple" groups="device" remote="sony" revision="r-mr1" />
+  <project path="device/sony/poplar" name="device-sony-poplar" groups="device" remote="sony" revision="r-mr1" />
+
+  <!--sdm845-->
+  <project path="device/sony/tama" name="device-sony-tama" groups="device" remote="sony" revision="r-mr1" />
+  <project path="device/sony/akari" name="device-sony-akari" groups="device" remote="sony" revision="r-mr1" />
+  <project path="device/sony/akatsuki" name="device-sony-akatsuki" groups="device" remote="sony" revision="r-mr1" />
+  <project path="device/sony/apollo" name="device-sony-apollo" groups="device" remote="sony" revision="r-mr1" />
+
+  <!--sm8150-->
+  <project path="device/sony/kumano" name="device-sony-kumano" groups="device" remote="sony" revision="r-mr1" />
+  <project path="device/sony/bahamut" name="device-sony-bahamut" groups="device" remote="sony" revision="r-mr1" />
+  <project path="device/sony/griffin" name="device-sony-griffin" groups="device" remote="sony" revision="r-mr1" />
+
+  <!--sm6125-->
+  <project path="device/sony/seine" name="device-sony-seine" groups="device" remote="sony" revision="r-mr1" />
+  <project path="device/sony/pdx201" name="device-sony-pdx201" groups="device" remote="sony" revision="r-mr1" />
+
+  <!--sm6350-->
+  <project path="device/sony/lena" name="device-sony-lena" groups="device" remote="sony" revision="r-mr1" />
+  <project path="device/sony/pdx213" name="device-sony-pdx213" groups="device" remote="sony" revision="r-mr1" />
+
+  <!--sm8250-->
+  <project path="device/sony/edo" name="device-sony-edo" groups="device" remote="sony" revision="r-mr1" />
+  <project path="device/sony/pdx203" name="device-sony-pdx203" groups="device" remote="sony" revision="r-mr1" />
+  <project path="device/sony/pdx206" name="device-sony-pdx206" groups="device" remote="sony" revision="r-mr1" />
+
+  <project path="vendor/nxp/" name="vendor-nxp" groups="device" remote="sony" revision="master" />
+  <project path="vendor/nxp/NXPNFCC_FW" name="nfc-NXPNFCC_FW" groups="device" remote="NXP" revision="master" />
+
+  <project path="vendor/oss/cash"  name="vendor-sony-oss-cash" groups="device" remote="sony" revision="master" />
+  <project path="vendor/oss/fingerprint"  name="vendor-sony-oss-fingerprint" groups="device" remote="sony" revision="master" />
+  <project path="vendor/oss/json-c"  name="json-c" groups="device" remote="sony" revision="master" />
+  <project path="vendor/oss/macaddrsetup" name="macaddrsetup" groups="device" remote="sony" revision="master" />
+  <project path="vendor/oss/opentelephony" name="SonyOpenTelephony" groups="device" remote="sony" revision="master" />
```

Comment:
Please switch this to r-mr1 as per upstream https://github.com/sonyxperiadev/local_manifests/commit/571b6b1af1db1ab30d82952c08545f248833b605